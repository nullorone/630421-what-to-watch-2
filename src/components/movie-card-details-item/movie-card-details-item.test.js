import React from "react";
import renderer from "react-test-renderer";
import MovieCardDetailsItem from "./movie-card-details-item";

describe(`Make snapshot`, () => {
  const initProps = {
    name: `My dad`,
    value: `Vladimir Putin`,
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <MovieCardDetailsItem {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
