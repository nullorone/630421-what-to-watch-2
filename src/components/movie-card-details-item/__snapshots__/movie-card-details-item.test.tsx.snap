// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<p
  className="movie-card__details-item"
>
  <strong
    className="movie-card__details-name"
  >
    My dad
  </strong>
  <span
    className="movie-card__details-value"
  >
    Vladimir Putin
  </span>
</p>
`;
